### DISCUSSION TITLE
Mounting with Docker Desktop (OSX)

### DISCUSSION THREAD

**Discussion by junkyardhokie** [philosowaffle/peloton-to-garmin#195] [Q&A]
How do you mount a custom configuration.local.json file when creating a container using the desktop app Docker Desktop (for Mac OSX)?

**Comment by cbordman** [comment]
use full paths. 
docker run --rm -v /Users/repos/peloton-to-garmin/configuration.local.json:/app/configuration.local.json -v /Users/repos/peloton-to-garmin/output:/app/output  --name peloton philosowaffle/peloton-to-garmin:vv2.2.0
